### PR TITLE
Specify files

### PR DESCRIPTION
**Pull request by hjdivad**
Also emit types.  Note that this isn't useful yet since the types aren't
exported within `main` (`dist/index`)

### CODE REVIEW

**Comment by hjdivad** [comment]
```
$ tar tfz malleatus-nyx-v0.2.0.tgz

package
package/CHANGELOG.md
package/LICENSE
package/RELEASE.md
package/dist
package/package.json
package/dist/commands
package/dist/index.d.ts
package/dist/index.js
package/dist/index.test.d.ts
package/dist/index.test.js
package/dist/commands/report-failure.d.ts
package/dist/commands/report-failure.js
package/dist/commands/report-failure.test.d.ts
package/dist/commands/report-failure.test.js

```

File: package.json [hunk 1]
```diff
@@ -52,6 +54,12 @@
     "ts-jest": "^25.2.1",
     "typescript": "~3.8.3"
   },
+  "files": [
+    "*.md",
+    "LICENSE",
+    "package.json",
+    "dist/*"
```
Comment (rwjblue) [inline package.json]:
```suggestion
    "dist/*",
    "!dist/**/*.test.js"
```

**Comment by hjdivad** [comment]
```
11:18:18 ❯ tar tfz malleatus-nyx-0.2.0.tgz  | sort
package/CHANGELOG.md
package/LICENSE
package/RELEASE.md
package/dist/commands/report-failure.d.ts
package/dist/commands/report-failure.js
package/dist/index.d.ts
package/dist/index.js
package/package.json
```

Note that this requires `npm pack`.  The exclusion pattern in `files` doesn't seem to work with `yarn pack`.

**Comment by hjdivad** [comment]
```
11:46:48 ❯ tar tfz malleatus-nyx-0.2.0.tgz | sort
package/CHANGELOG.md
package/LICENSE
package/dist/commands/report-failure.d.ts
package/dist/commands/report-failure.js
package/dist/index.d.ts
package/dist/index.js
package/package.json
```

**Pull request merged**